src/components/Header.tsx

```diff
@@ -29,6 +30,11 @@ const options: Options = {
 		"forest",
 		"rose-milk",
 	],
+	theme: [
+		"words",
+		"sentances",
```

Comment:
Fix spelling of "sentences"

Reply:
yeah sorry, im not a good speller... should be fixed now.